email/sqr-submission.html

```diff
@@ -1,6 +1,55 @@
-<b>Current set:</b>
-<div style="font-family: Courier,serif">
-  Please update your previous submission using this update link: http://purebhakti.info/audioseva/form/sound-quality-report?token={{params.submission.token}}.<br>Please note &#8212; do not make a new submission using this link. For a new submission, use the link in the allotment email.<br>
+<body>
+  <div style="color:red; font-weight:bold;">
+    {% if params.submission.cancellation.notPreferredLanguage %}
+      <p>The lecture is not in devotee's preferred language!</p>
+    {% endif %}
+
+    {% if params.submission.cancellation.audioProblem %}
+      <p>Unable to play or download the audio {{ params.submission.comments }}</p>
+    {% endif %}
+
+    {% if params.submission.changed != params.submission.completed %}
+      <p>This is an updated submission!</p>
+    {% endif %}
+
+    {% if params.fileData.soundQualityReporting.assignee.emailAddress != params.submission.author.emailAddress %}
+      <div>
+        <p>
+          File is alloted to another email
+          id - {{ params.fileData.soundQualityReporting.assignee.emailAddress }}!
+        </p>
+
+        <p>Status of file is {{ params.fileData.soundQualityReporting.status }}</p>
+
+        <p>Audio file name {{ params.submission.fileName }} is not found in the Backend!</p>
+      </div>
+    {% endif %}
+
+    {% if params.fileData.soundQualityReporting.status == "Given" %}
+      <p>It's time to allot!</p>
```

Comment:
You know, now I'm thinking that it might be better to produce all the warnings in the function and pass the warnings as `params.warnings`. This way it will be more convenient to add new warnings later without changing the template.

So please move all this logic into the function.